lib/conjur/trusted_proxy_filter.rb

```diff
@@ -29,25 +29,25 @@ def validate_trusted_proxies
       trusted_proxies
     end
 
-    def trusted_proxies    
+    def trusted_proxies
       return @cached_trusted_proxies if @cached_trusted_proxies
 
       # The trusted proxy IPs are `127.0.0.1` plus those defined in the
       # `TRUSTED_PROXIES` environment variable.
-      proxy_ips = [IPAddr.new('127.0.0.1')] + env_trusted_proxies
-      
+      proxy_ips = [IPAddr.new('127.0.0.1')] + configured_trusted_proxies
+
       # If not disabled, cache the IP address list
       @cached_trusted_proxies = proxy_ips unless @options[:disable_cache]
 
       proxy_ips
     end
 
-    # Reek flags @env['TRUSTED_PROXIES'] as :reek:DuplicateMethodCall. Refactoring
-    # this would not enhance the readability or performance.
-    def env_trusted_proxies
-      return [] unless @env['TRUSTED_PROXIES']
+    def configured_trusted_proxies
```

Comment:
Optional idea, I throw it out for your consideration -- get rid of helper method:

```ruby
def configured_trusted_proxies
  trusted_proxies = @config.trusted_proxies

  return [] unless trusted_proxies

  Set
    .new(trusted_proxies.split(','))
    .map { |cidr| IPAddr.new(cidr) }
rescue IPAddr::Error
  raise Errors::Conjur::InvalidTrustedProxies, cidr
end
```